File: android/src/androidTest/java/TheMainActivity.java
import android.support.test.rule.ActivityTestRule;
import com.jraska.falcon.FalconSpoon;
import org.junit.Rule;
import org.junit.Test;
import org.ligi.scr.MainActivity;

public class TheMainActivity {

    @Rule
    public ActivityTestRule<MainActivity> rule = new ActivityTestRule<>(MainActivity.class);

    @Test
    public void testMainActivityStarts() {

        FalconSpoon.screenshot(rule.getActivity(), "main");
    }
}

File: android/src/androidTest/java/TheMainActivity.kt
import com.jraska.falcon.FalconSpoon
import org.junit.Rule
import org.junit.Test
import org.ligi.scr.MainActivity
import org.ligi.trulesk.TruleskActivityRule


class TheMainActivity {

    @get:Rule
    var rule = TruleskActivityRule(MainActivity::class.java)

    @Test
    fun testMainActivityStarts() {

        FalconSpoon.screenshot(rule.activity, "main")
    }
}
